helix-front/angular.json

```diff
@@ -16,7 +16,12 @@
             "outputPath": "dist/public",
             "assets": ["client/assets/", "client/favicon.ico"],
             "scripts": [],
-            "styles": ["client/styles.scss", "client/theme.scss"]
+            "styles": ["client/styles.scss", "client/theme.scss"],
+            "stylePreprocessorOptions": {
+              "includePaths": [
+                "client/"
+              ]
+            }
```

Comment:
This change is recommended by this [StackOverflow Answer from earlier today](https://stackoverflow.com/a/71922508/1732222). 


However, I still see this error: 

```bash
./client/main.ts - Error: Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
Error: Transform failed with 1 error:
helix/helix-front/client/app/cluster/cluster-list/cluster-list.component.scss:17:100: ERROR: Unterminated string token
    at helix/helix-front/node_modules/@ngtools/webpack/src/ivy/loader.js:81:18
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```